install/install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#! /usr/bin/env sh
```

Comment:
Is the space here intentional? I think some systems could have issues with that.

Reply:
It was intentional and I always do it, but I never knew it could be an issue. Apparently it's a great debate: https://web.archive.org/web/20170215070707/http://www.in-ulm.de/~mascheck/various/shebang/#details

I have no issue with removing the space; and will do!

Reply:
done

Reply:
`set -o pipefail` immediately exists regardless of any attempt to ignore the exit code with `|| true` in dash... simple test:

```sh
temp=$(mktemp -d)
cat << EOT > "$temp"/with-pipefail.sh
#! /usr/bin/env sh

set +e
printf "%s\n" "attempting to set pipefail"
set -o pipefail 2>/dev/null || true
printf "%s\n" "we got here"
EOT

chmod +x "$temp"/with-pipefail.sh

docker run --volume="$temp":/test debian /test/with-pipefail.sh
rm -rf "$temp"
```